src/styles/cylc/_markdown.scss

```diff
@@ -1,33 +1,180 @@
-/**
- * Copyright (C) NIWA & British Crown (Met Office) & Contributors.
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
-.markdown {
-  p, ul, ol {
-    &:not(:last-child) {
-      margin-bottom: 0.8em;
-    }
-  }
-  ul, ol {
-    padding-left: 24px;
-  }
-  .v-theme--light & code {
-    background-color: rgba(0, 0, 0, 0.05);
-  }
-  .v-theme--dark & code {
-    background-color: rgba(255, 255, 255, 0.1);
-  }
```

Comment:
We could look at more carefully incorporating the old styling into the new, as it was chosen to ensure the limited Markdown we already had fits with Vuetify